tests/manage/pv_services/test_pvc_delete_subvolumegroup.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+import logging
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pvc import delete_pvcs
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+from ocs_ci.framework.testlib import ManageTest, skipif_ocs_version, polarion_id
+from ocs_ci.utility.utils import TimeoutSampler
+
+logger = logging.getLogger(__name__)
+
+
+@skipif_ocs_version("<4.12")
+@polarion_id("OCS-4664")
+class TestPvcDeleteSubVolumeGroup(ManageTest):
+    """
+    Delete PVC subvolumegroup and make sure PVC reaches bound state
+    """
+
+    subvolumegroup_ls_cmd = (
+        "ceph fs subvolumegroup ls ocs-storagecluster-cephfilesystem"
+    )
+
+    @pytest.fixture(autouse=True)
+    def setup(self, project_factory):
+        """
+        create resources for the test
+
+        Args:
+            project_factory: A fixture to create new project
+        """
+        logger.info("Creating project")
+        self.proj_obj = project_factory()
+        # Get ceph tool pod
+        self.tool_pod = get_ceph_tools_pod()
+
+    def is_volumegroup_empty(self):
+        """
+        Check if volumegroup is empty
+
+        Returns:
+            bool: True if empty, False otherwise
+        """
+        out = self.tool_pod.exec_ceph_cmd(
+            ceph_cmd="ceph fs subvolume ls ocs-storagecluster-cephfilesystem csi",
+            format=None,
+        )
+        return not bool(out)
+
+    def test_pvc_delete_subvolumegroup(self, pvc_factory):
+        """
+        Delete PVC subvolumegroup and make sure PVC reaches bound state
+        """
+
+        logger.info("Creating pvc")
+        pvc_obj = pvc_factory(
+            interface=constants.CEPHFILESYSTEM,
+            size=5,
+            project=self.proj_obj,
+        )
+        logger.info(f"Deleting pvc {pvc_obj.name}")
+        delete_pvcs(pvc_objs=[pvc_obj])
+
+        # Make sure volumegroup is empty before deletion
+        for sampler in TimeoutSampler(
+            timeout=60, sleep=5, func=self.is_volumegroup_empty
+        ):
+            if sampler:
+                logger.info("volumegroup is empty!")
+                break
+
+        # Remove subvolumegroup
+        self.tool_pod.exec_ceph_cmd(
+            ceph_cmd="ceph fs subvolumegroup rm ocs-storagecluster-cephfilesystem csi",
+            format=None,
+        )
+
+        # List subvolumegroup (should be empty)
+        subvolume = self.tool_pod.exec_ceph_cmd(
+            ceph_cmd=self.subvolumegroup_ls_cmd, format=None
+        )
+        assert not subvolume, "Subvolumegroup should be empty"
+
```

Comment:
Please add a subvolume group empty success message in the log, these info can be helpful when debugging the test failure

Reply:
Done